extension/EdFi.Ods.Extensions.LMSX/LMS-Harmonizer/1080-lms-assignment.sql

```diff
@@ -7,26 +7,25 @@ CREATE OR ALTER PROCEDURE [lms].[harmonize_assignment] @SourceSystem nvarchar(25
 BEGIN
     SET NOCOUNT ON;
 
-
 	-- Load temporary table
 	SELECT
-		lmsAssignment.AssignmentIdentifier [AssignmentIdentifier],
-		sourceSystemDescriptor.DescriptorId [LMSSourceSystemDescriptorId],
-		lmsAssignment.[Title] [Title],
-		assignmentCatDescriptor.DescriptorId [AssignmentCategoryDescriptorId],
-		lmsAssignment.[AssignmentDescription] [AssignmentDescription],
-		lmsAssignment.[StartDateTime] [StartDateTime],
-		lmsAssignment.[EndDateTime] [EndDateTime],
-		lmsAssignment.[DueDateTime] [DueDateTime],
-		lmsAssignment.[MaxPoints] [MaxPoints],
-		edfiSection.SectionIdentifier [SectionIdentifier],
-		edfiSection.LocalCourseCode [LocalCourseCode],
-		edfiSection.[SessionName] [SessionName],
-		edfiSection.[SchoolYear] [SchoolYear],
+		lmsAssignment.SourceSystemIdentifier as AssignmentIdentifier,
```

Comment:
I like to have `as` - although it is unnecessary, I find it easier to read: the syntax highlighting helps me see that an alias is being created.

Avoiding brackets whenever possible makes it easier to port the code to PostgreSQL in the future. And it is easier to write without brackets.

Reply:
Oh, and this line 12 is the real reason that I was modifying the file. Note that it is now pulling from `SourceSystemIdentifier` instead of `AssignmentIdentifeir`.

Reply:
yes it is better, I wrote the harmonize for the assignments and cannot remember why I used the assignment identifier instead of the source system identifier

Reply:
Easy mistake to make, as we mixed up the column names: in one place `AssignmentIdentifier` is an auto-increment int, and in another place it is a varchar. But that's not obvious just from the column name.